velox/docs/functions/presto/binary.rst

```diff
@@ -75,10 +75,30 @@ Binary Functions
 
     Returns the length of ``binary`` in bytes.
 
+.. function:: lpad(binary, size, padbinary) -> varbinary
+    :noindex:
+    
+    Left pads ``binary`` to ``size`` bytes with ``padbinary``.
+    If ``size`` is less than the length of ``binary``, the result is
+    truncated to ``size`` characters. ``size`` must not be negative
+    and ``padbinary`` must be non-empty. ``size`` has a maximum value of 1 MiB.
+    In the case of ``size`` being smaller than the length of ``binary``, 
```

Comment:
Thank you for adding this clarifying. Would be nice to also include an example in case a reader things that 'right' here is a typo. We may need to update docs for lpad(varchar) as well.